integration/utils.go

```diff
@@ -17,6 +17,10 @@ import (
 const skopeoBinary = "skopeo"
 const decompressDirsBinary = "./decompress-dirs.sh"
 
+const testFQIN = "docker://quay.io/libpod/busybox" // tag left off on purpose, some tests need to add a special one
+const testFQIN64 = "docker://quay.io/libpod/busybox:amd64"
+const testFQINml = "docker://quay.io/libpod/alpine_nginx:master" // multi-layer
```

Comment:
This variable name is a bit too much of a puzzle.

`multiLayerTestFQIN`? `testFQINMultiLayer`?

Reply:
Good suggestion, okay willdo.